SamplesDashboard/SamplesDashboard/ClientApp/src/views/Home.tsx

```diff
@@ -1,10 +1,18 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Repositories from './Repositories';
 import { PivotItem, Pivot, PivotLinkSize, FontSizes } from 'office-ui-fabric-react';
+import queryString from 'query-string';
 
 
-class Home extends Component {          
-    public render() {
+export default class Home extends React.Component<any, any> {
```

Comment:
>any, any [](start = 50, length = 8)

can we use types where we can so it's more typescript than javascript.  It also makes things more long term maintainable. it doesn't look like the component has state so I don't think you need the state type.